hadoop-ozone/dev-support/checks/_mvn_unit_report.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## generate summary txt file
+find "." -name 'TEST*.xml' -print0 \
+    | xargs -n1 -0 "grep" -l -E "<failure|<error" \
+    | awk -F/ '{sub("'"TEST-JUNIT_TEST_OUTPUT_DIR"'",""); sub(".xml",""); print $NF}' \
+    | tee "$REPORT_DIR/summary.txt"
+
+
+#Collect of all of the FAILED results
+for dir in $(find "." -name surefire-reports); do
+   for file in $(grep -l -r FAILURE --include="*.txt" $dir | grep -v output.txt ); do
```

Comment:
shellcheck:16: note: To read lines rather than words, pipe/redirect to a 'while read' loop. [SC2013]
shellcheck:55: note: Double quote to prevent globbing and word splitting. [SC2086]